BUILD.bazel

```diff
@@ -0,0 +1,297 @@
+load(
+    "//ign_bazel:build_defs.bzl",
+    "IGNITION_ROOT",
+    "IGNITION_VISIBILITY",
+    "cmake_configure_file",
+    "generate_include_header",
+    "generate_yaml",
+)
+load(
+    "@bazelruby_rules_ruby//ruby:defs.bzl",
+    "ruby_binary",
+    "ruby_library",
+    "ruby_rspec",
+    "ruby_test",
+)
+
+package(
+    default_visibility = IGNITION_VISIBILITY,
+    features = [
+        "-parse_headers",
+        "-layering_check",
+    ],
+)
+
+licenses(["notice"])
+
+exports_files(["LICENSE"])
+
+PROJECT_NAME = "sdformat"
+
+SDF_MAJOR_VERSION = 10
+
+SDF_MINOR_VERSION = 0
+
+SDF_PATCH_VERSION = 0
```

Comment:
are we adding CI to confirm that these versions stay synchronized with cmake?

Reply:
I am adding a script to the bazel-ci repo to verify that this stays in sync with the most recent git tag on the branch, for example: `sdformat10_10.1.0`